src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py

```diff
@@ -39,7 +40,8 @@ def get_valid_os(cmd):
 
 
 def get_valid_architecture(cmd):
-    Architecture = cmd.get_models('Architecture')
+    from .sdk.models import Architecture
+    # Architecture = cmd.get_models('Architecture')
```

Comment:
Architecture is commented

Reply:
Yes, waiting for the new SDK to be merged. Once thats done, I will not use local SDK.